tests/unit_tests/storage/test_storage_facility.cu

```diff
@@ -0,0 +1,11 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2019, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "test_storage_facility.cpp"
```

Comment:
I didn't have a better idea... Let me know if you have one.

Reply:
I think that  you don't need dummy `.cu` file here. `gridtools_add_unit_test` will do normalization

Reply:
I tried it, but it didn't work. I can try again, because maybe I got lost in the options...

Reply:
You are right, I got lost in the dependencies...